expyfun/stimuli/_tracker.py

```diff
@@ -239,6 +240,20 @@ def respond(self, correct):
                 if self._direction <= 0:
                     self._direction = 1
 
+        if self._repeat_limit == 'reversals':
+            if self._x[-1] == self._x_min:
+                step_dir = 1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+            if self._x[-1] == self._x_max:
+                step_dir = -1
+                self._n_up = 0
+                reversal = True
+                self._n_reversals += 1
+                self._direction = 1
+
```

Comment:
@Eric89GXL @drammock @rkmaddox 

There should probably be some warning when the data is doing this. But I can think of a couple of approaches...
1. Be obnoxious and warn user _every_ time they hit a floor/ceiling.
2. Warn the user that a floor/ceiling was hit x times (no matter how big or small x is) when a tracker stops.
3. Warn the user that "enough" floor/ceiling hits occurred when a tracker stops--then again, how does one decide what "enough" is?
I don't think that there are scenarios in which I'd want to abort an experiment because of floor/ceiling hits.